x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/behavior.tsx

```diff
@@ -25,14 +25,17 @@ import { SecurityPageName } from '../../../../../../app/types';
 export const BehaviorProtection = React.memo(() => {
   const OSes: Immutable<BehaviorProtectionOSes[]> = [OS.windows];
   const protection = 'behavior_protection';
+  const protectionLabel = 'behavior';
```

Comment:
I'm not sure... Should be this behind i18n?

Reply:
I don't think so. this is a noun for the specific name of the protection. In the other protections the name is used as is.

@kevinlog should we start translating the name of the protections? if so, we'd have to set that up also for malware and ransomware.

Reply:
we only need to translate user readable text in the UI.   If these are used as variable names in code for identifying protections, no need to translate

Reply:
Great. so @dasansol92 we don't need to translate those variables

Reply:
But is this the name will be displayed in the protections form? So it is a user readable text right?

Reply:
It is a variable that is later output here: https://github.com/elastic/kibana/blob/cbcf63d3768a0e3801d8561baaf98d13a37c6ede/x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/protection_switch.tsx#L100

`{protectionName} protections {mode, select, true {enabled} false {disabled}}`

What we can do is make this label to have the "protections" string and use it here as an already translated Noun:

`{protectionName} {mode, select, true {enabled} false {disabled}}`

I can do that and modify the other protections that are not using a label.

Reply:
Ahá, I see. Then it's ok for me! Thanks for clarify this!!

Reply:
I'll do the change to include the `protections` part in this and the other protections. It will provide a more consistent UX in other languages.

Reply:
@dasansol92 I pushed the changes updating all the other protections too to use a translated string to pass.